.eslintrc.json

```diff
@@ -2,6 +2,67 @@
   "extends": [
     "@pixi/eslint-config"
   ],
+  "parserOptions": {
+    "project": ["./tsconfig.eslint.json"]
+  },
+  "overrides": [
+    {
+      "files": ["*.ts"],
+      "extends": ["plugin:@typescript-eslint/recommended"],
+      "rules": {
+        "@typescript-eslint/no-explicit-any": 0,
+        "@typescript-eslint/interface-name-prefix": 0,
+
```

Comment:
These changes will be removed if the eslint PR get merged

Reply:
Made version v4.0.0 of eslint-config with these changes. Can you update?

Reply:
Updated!